www/js/app.js

```diff
@@ -358,7 +358,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         return false;
     });
     $('#btnTop').on('click', function() {
-        $("#articleContent").contents().scrollTop(0);
+        $('#articleContent').contents().scrollTop(0);
```

Comment:
Ditto?

Reply:
Can do this one in this PR. I'll get on to it.